code/modules/paperwork/paper.dm

```diff
@@ -348,6 +348,11 @@
 	..()
 	update_icon()
 
+/obj/item/weapon/paper/experience_pressure_difference(pressure_difference, direction)
+	if(last_high_pressure_movement_air_cycle < SSair.times_fired)
+		last_high_pressure_movement_air_cycle = SSair.times_fired
+		step(src, direction)
+
```

Comment:
oh, at first I was worried about divide by zero memes, but i added a check anyways, so i guess this could be removed.